packages/ui/src/routes/+page.svelte

```diff
@@ -10,13 +10,55 @@
 	import { posts } from '$lib/stores/post'
 	import { profile } from '$lib/stores/profile'
 	import { goto } from '$app/navigation'
+	import { browser } from '$app/environment';
 	import Masonry from '$lib/masonry.svelte'
 
-	let y: number
+	let windowWidth: number = browser ? window.innerWidth : 0
+
+	function GetMasonryColumnWidth(w: number) {
+
+		if (w < 739) {
+			return '100%'
+		}
+
+		else if (w < 1060) {
+			return 'minmax(min(100%/2, max(320px, 100%/2)), 1fr)'
+		}
+
+		else if (w < 1381) {
+			return 'minmax(min(100%/3, max(320px, 100%/3)), 1fr)'
+		}
```

Comment:
Here it is not necessary to use `else if` because the previous `if` statements return from the function and the execution of the function therefore does not continue.